later/lib/presentation/cubit/home_cubit/home_cubit.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:injectable/injectable.dart';
+
+import 'package:later/domain/usecases/watch_all_records_use_case.dart';
+import 'home_state.dart';
+
+@lazySingleton
+class HomePageCubit extends Cubit<HomeState> {
+  HomePageCubit({required this.useCase}) : super(HomeEmpty());
+
+  final WatchAllRecordsUseCase useCase;
```

Comment:
1. Make it private
2. Name it `watchAllRecordsUseCase` so you clearly know what is this use case. In the future we might have other use cases in this cubit.